docs/howto/platform-sdk-quickstart-guide.md

```diff
@@ -0,0 +1,194 @@
+# Platform SDK Quickstart Guide
+
+#### Python SDK v2
+
+Version 2 of AYX Python SDK is now available. For the best experience,
+and to ensure compatibility with the most recent versions of Designer,
+please use v2 of [AYX Python
+SDK.](https://help.alteryx.com/developer-help/ayx-python-sdk-v2)
```

Comment:
Point this to the new reference on ayx-developer-sdk repo.

This should point to the corresponding markdown version